yarn-project/sequencer-client/src/sequencer/public_processor.ts

```diff
@@ -90,9 +117,9 @@ export class PublicProcessor {
     const callStackItem = new PublicCallStackItem(contractAddress, txRequest.functionData, publicCircuitOutput);
     const publicCallStackPreimages: PublicCallStackItem[] = times(PUBLIC_CALL_STACK_LENGTH, PublicCallStackItem.empty);
 
-    // TODO: Get these from the ContractDataSource once available
-    const portalContractAddress = Fr.random();
-    const bytecodeHash = Fr.random();
+    // TODO: How to get the bytecode hash? Pedersen or SHA256? What generator to use?
+    const bytecodeHash = Fr.fromBuffer(pedersenGetHash(await CircuitsWasm.get(), functionBytecode.bytecode));
```

Comment:
@iAmMichaelConnor how should we calculate the bytecode_hash of the PublicCallData in the PublicKernelInputs? Pedersen with a new generator?